recipes/ipytree/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "ipytree" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "dbee2f37af813338c59db79f2050f6b082b983d7d58041e092fa69d5580f4ef8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
```

Comment:
`build:` should be changed to `host:` here, look here: https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip